device/isis/routeradvanced.yaml

```diff
@@ -0,0 +1,71 @@
+components:
+  schemas:
+    Isis.Advanced:
+      description: |-
+        Contains ISIS router advanced properties.
+      type: object
+      properties:
+        enable_hello_padding:
+          description: >-
+            It enables padding of Hello message to MTU size.
+          type: boolean
+          default: true
+        max_area_addresses:
+          description: >-
+            The Number of Area Addresses permitted, with a valid range from 0 to 254. 
+            A zero indicates a maximum of 3 addresses. 
+          type: integer
+          minimum: 0
+          maximum: 254
+          default: 3
+        area_addresses:
```

Comment:
Session will come up even if area addresses are not configured since by default we will accept any area address sent by the DUT. By default we will always include area address "490001" in initial Hello,  which if required,  user can modify by setting this field.